### PR TITLE
Balance tactical helmets

### PR DESCRIPTION
**Pull request by DericHuynh**
#### Summary

SUMMARY: Balance "Balance Tactical Helmets"

#### Purpose of change

The tactical helmets were essentially useless before. They boasted defensive stats only a 1/3 of the army helmet, while also being relatively rare in comparison. A quick search on google showed that the tactical helmets and army helmets are essentially the same, so I've decided to buff the Tactical helmets in order to make them a viable alternative.

#### Describe the solution

I balanced the Tactical helmets in terms of weight, volume, encumbrance, and protection value, while also giving them the same flags as the army helmet. This should make the tactical helmets comparable.

#### Describe alternatives you've considered

I've considered nerfing the army helmet, but reminded myself of the ESAPI vest and decided not to.

#### Testing

I ran the game and spawned them in to test if the JSON worked.

### CODE REVIEW

File: data/json/items/armor/helmets.json [hunk 1]
```diff
@@ -77,10 +77,10 @@
...
     "name": { "str": "tactical full helmet" },
     "description": "An all-encompassing black helmet that covers your entire face and neck, providing excellent protection from all sorts of damage.",
-    "weight": "1324 g",
-    "volume": "2 L",
-    "price": 50000,
-    "price_postapoc": 1000,
+    "weight": "1594 g",
+    "volume": "2250 ml",
+    "price": 25000,
+    "price_postapoc": 2500,
```
Comment (Night-Pryanik) [inline data/json/items/armor/helmets.json:83]: I guess this is wrong. Sturdy helmets should be one of the first things to seek after the apocalypse, and thus they shouldn't cost mere $25. I'd set the `price_postapoc` to at least of its pre-apocalypse value, or maybe even raise it a bit.

Comment (Xaleth) [reply data/json/items/armor/helmets.json:83]: also while you're at it, make it a string :pray:

Comment (DericHuynh) [reply data/json/items/armor/helmets.json:83]: Strange, the army helmet was listed as 1500 postapoc.

Comment (DericHuynh) [reply data/json/items/armor/helmets.json:83]: I've looked through the other items, and it seems that the postapoc pricing is relative. For example, the A7 Laser rifle, a more endgame item is 7500 postapoc, or 75$.

Comment (DericHuynh) [reply data/json/items/armor/helmets.json:83]: > 
> 
> also while you're at it, make it a string 🙏

what do you mean "make it a string"

Comment (Night-Pryanik) [reply data/json/items/armor/helmets.json:83]: We need a total overhaul of postapoc prices, so we can start with this helmet.

**Comment by Kelenius** [comment]
> A quick search on google showed that the tactical helmets and army helmets are essentially the same

What's the actual difference between them? Maybe they could be simply merged into a single item if they're so similar.

**Comment by Zireael07** [comment]
@Kelenius: A quick google shows me this:
Screenshot_20201124_135125
Screenshot_20201124_135108

So tactical helmet > army helmet because tactical helmet can include a face protection, while army helmet does not have face protection at all. For CDDA purposes, let's say ALL tac helmets have face protection.

**Comment by Kelenius** [comment]
@Zireael07 

The one covering the face would be `tac_fullhelmet`, "An all-encompassing black helmet that covers your entire face and neck, providing excellent protection from all sorts of damage.". The game also has `tac_helmet`, "A lightweight black helmet that provides excellent protection from all sorts of damage." which does not cover the face.

**Comment by Zireael07** [comment]
In that case, I would indeed merge army helmet and tac_helmet (the one without face protection) into one.

**Comment by Rivet-the-Zombie** [comment]
>In that case, I would indeed merge army helmet and tac_helmet (the one without face protection) into one.

It would still be good to keep separate items - even if one's stats are just copied directly from the other - so that the stuff like the various police professions and zombies don't suddenly start showing up wearing army helmets.

**Pull request merged**